bin/lock-wikis

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Question: Is there a reason to explicitly use `bash` when no bashisms are used in the scripts? I.e. would `/bin/sh` also do and allow for greater portability?

Reply:
Makes sense to me. Problem is that I always have been using bash for scripting so I don't know what is missing with sh :D

Reply:
The one major difference I have encountered is that bash knows `[[` when sh only knows `[`. Also `sh` doesn't know about `set -o pipefail` as well as `getopts` (see below).